src/data/contributors/NikhilSharmaWe.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Nikhil Sharma",
+  "skills": ["Python", "C++", "JS"],
+  "cause": "learn while contributing to the world through open source",
+  "image": "Avatar1",
+  "github": "https://github.com/NikhilSharmaWe",
```

Comment:
please remove comma from the last array item (github). It is catched by eslint. and causing the build to fail.